src/test/kotlin/pl/edu/uj/ii/ksi/mordor/events/EmailVerificationProcessorTest.kt

```diff
@@ -12,24 +13,26 @@ import pl.edu.uj.ii.ksi.mordor.persistence.entities.EmailVerificationToken
 import pl.edu.uj.ii.ksi.mordor.persistence.entities.Role
 import pl.edu.uj.ii.ksi.mordor.persistence.entities.User
 import pl.edu.uj.ii.ksi.mordor.persistence.repositories.EmailVerificationTokenRepository
-import javax.mail.internet.MimeMessage
 
 class EmailVerificationProcessorTest {
+    companion object {
```

Comment:
@apardyl shouldn't this be exported to some kind of configuration class/file?

Reply:
erm, that's test data xD

Reply:
@apardyl nevermind, not really sure how I  missed this.